.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
We can add a .gitignore entry for this (since half the team seems to use VSCode),
but it does no harm, so not that important.